test/TestAsset/http-status-codes.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Where does this file come from? And can we instead request it via HTTPS? That's what I was asking for :-)